TV Shows/Movies: Document theme media behaviour

mihawk90 opened the pull request:
This is a followup to jellyfin/jellyfin-web#5714 to document those changes.

* User setting was undocumented
* Proper example boxes like in other sections
* New random media playback
* Theme Videos preferred over Theme Music
* Recommendations for theme media video format
* Per Season theme media was undocumented
* Movie theme media was entirely undocumented

mihawk90 commented:
I meant to mark this as a draft because the web PR isn't released yet anyway 😅

mihawk90 commented:
Rebased and Preview looks fine at first glance, unless you want the example boxes in the include file as well.

mihawk90 commented:
Like this? :thinking: 

Or did you mean combining the `Videos` and `Music` header as well?

mihawk90 commented:
OK, this should be it then.

I moved it under `Extras`, but I left it as a separate file, just moved the include.

Also fixed the linting error.

BotBlake commented on docs/general/server/media/_video-theme-media.md:
```diff
@@ -0,0 +1,73 @@
+
+In case both Theme Videos and Theme Music are found, Theme Videos will be preferred when enabled and Theme Music will not play.
+
+In the WebUI and WebUI-based clients, if there are multiple theme media found, they will be shuffled when opening the listing. [This cannot be changed](https://github.com/jellyfin/jellyfin-web/pull/5714).
```
Personaly, I am not a fan of the link here.
```suggestion
In the WebUI and WebUI-based clients, if there are multiple theme media found, they will be shuffled when opening the listing. This cannot be changed.
```

BotBlake commented on docs/general/server/media/_video-theme-media.md:
```diff
@@ -0,0 +1,73 @@
+
+#### Videos
+
+- backdrops/\*
```
These rules are super basic, I would suggest to not have individual chapters for Videos, Movies and Example - Instead this could easily fit into the text above.

mihawk90 replied:
I just followed the rest of the page to have it consistently layed out. I don't think having it in the text has any benefit, on the contrary I feel like it would get lost and/or overread. When I am looking for just the paths used I look for the list, not somewhere buried in the text.

BotBlake commented on docs/general/server/media/_video-theme-media.md:
```diff
@@ -0,0 +1,73 @@
+
+Theme media gives browsing your library a more audio-visual touch by playing theme songs and/or theme videos in the background while you're looking at your media listings.
+
+:::tip
```
Try to keep the admonition count as low as possible. Having to many in the documentation will make them lose their effect on raising attention!
If something is not IMPORTANT, we dont realy want admonition blocks to be used.

BotBlake reviewed (changes requested):
Hi there!
Is this still in Draft? The WebPR has been merged.

Let me ad some feedback as well, while I am at it:
We generally try to keep the amount of admonitions as low as possible.
<img width="1463" height="1099" alt="grafik" src="https://github.com/user-attachments/assets/36b2d4cb-f92b-48e4-8d22-98f30bf35de7" />
This is defenetly a few to many. If possible, please remove a few of those (actually, I think all of them can be removed here)

Additionaly, I think this is way too long to be an reusable element! If possible, try to squeeze down the chapter count. I added some feedback regarding that in the comments.

Thanks for the contribution!
Kind regards

mihawk90 commented:
> Is this still in Draft? The WebPR has been merged.

Merged, but not released. This is in draft because it does not apply to the current released version of JF.

> This is defenetly a few to many. If possible, please remove a few of those (actually, I think all of them can be removed here)

I agree with that and that bothered me too while I wrote it, but I felt like all of these are important.

I guess the second one could be removed because the transcode is handled automatically and the transcoding thing under videos implies this anyway.

But the others I feel like will come up  with users because theme media is not enabled by default and the question of "how does multiple different theme media work" is valid too. IDK, maybe have that one just in the text?

I would certainly not remove the `Tip` boxes tho.

mihawk90 commented:
Made some changes as per comments above:

* first "Note" removed
* changed second "Note" to a regular paragraph
* PR link removed

Also, two small things in addition:
* changed "Theme Music" to "Theme songs", as per the client setting
* swapped Music/Songs and Video chapters to space out the boxes further and for consistency within the text

felix920506 commented:
@mihawk90 we track things that don't apply to the current version using the future-release tag. If the content is good please mark this ready to review.
Please also resolve the merge conflicts.

mihawk90 commented:
I just stumbled on this while I was looking at the network requests for something unrelated...

```json
{
    "ThemeVideosResult": {
        "OwnerId": "f27caa37e5142225cceded48f6553502",
        "Items": [],
        "TotalRecordCount": 0,
        "StartIndex": 0
    },
    "ThemeSongsResult": {
        "OwnerId": "f27caa37e5142225cceded48f6553502",
        "Items": [],
        "TotalRecordCount": 0,
        "StartIndex": 0
    },
    "SoundtrackSongsResult": {
        "OwnerId": "00000000000000000000000000000000",
        "Items": [],
        "TotalRecordCount": 0,
        "StartIndex": 0
    }
}
```

`ThemeVideosResult` and `ThemeSongsResult` is obvious, but what exactly is `SoundtrackSongsResult`? What is the difference to the other two and why does it have a dummy(?) `OwnerId`?

Is this an emby leftover that noone knows about or something new that's undocumented as well?

It's mentioned in the result in the [API docs](https://api.jellyfin.org/#tag/Library/operation/GetThemeMedia) as well (although with a proper `OwnerId`), but no further details on it.

If anyone can enlighten me, I can add it to the PR as well.

PS:
Since 10.11 is out now the `future-release` label can be removed?
